### ISSUE TITLE
SUGGESTION: Add world ping display

### ISSUE THREAD

**Issue by colinberan** [jcarbelbide/tog-crowdsourcing#2]
Would be nice to see my connection to each world as well so I can decide which one would be best to connect to.

**Comment by jcarbelbide** [comment]
I pretty much copy pasted the sidebar code from the World Selector plugin. I'm not at my computer right now, but I think that plugin has ping right? I might have replaced the ping column with hits or something because I was worried about space, or it was just easier that way. Can't remember, but I can check it out and see if it's simple to add it in 

I think when I first made this, I never imagined there would be so many choices for worlds to connect to, so I didn't think ping would matter haha :)

**Comment by 67protons** [comment]
Ping on this issue. The plugin still seems to be missing this and it'd be great to have this feature. I think there should be enough room for a ping column while keeping the hits column as well as the World Selector plugin is able to accommodate 4 columns.

I'd be willing to take a stab at it as well if you're too busy

**Comment by jcarbelbide** [comment]
Hey @67protons, I might take you up on that offer. Let me look at this over the weekend to see what would be needed. I've been pretty busy at work lately so I haven't had time to look at this

**Comment by jcarbelbide** [comment]
Had no time this weekend, but should be able to take a look this weekend.

**Comment by jcarbelbide** [comment]
Currently working on this. Will push a branch when I have time

**Comment by jcarbelbide** [comment]
PR open. Will leave it open for a day and will then create a PR for the plugin hub

https://github.com/jcarbelbide/tog-crowdsourcing/pull/9

**Comment by jcarbelbide** [comment]
Yeah, good point. I had a similar thought this morning haha. I'll see if I can borrow some space from the other columns

Thanks for the suggestion about ping sort. I believe can incorporate a ping sort as well. I can work on that today or tomorrow 👍

**Comment by jcarbelbide** [comment]
Apologies for the delay, and thank you for the patience! Got busy during the week, but I just found some time and adjusted it per your recommendations. Great suggestions, thank you!

Each column has enough space to show each header, and instead of sorting by:
```Stream Order --> Hits --> World #```

now we sort by:
```Stream Order --> Ping --> Hits --> World #```

I think this is much better :)

![image](https://github.com/jcarbelbide/tog-crowdsourcing/assets/19232293/0e2ae8b4-d276-4346-9533-914c3c3e9689)

I'll leave this open for a bit too to get some feedback, then I'll open a PR in the plugin hub repo

**Comment by 67protons** [comment]
This looks awesome! Can't wait for it to go live!

**Comment by jcarbelbide** [comment]
Small issue merging this feature - the maintainers brought up a good point that they do not want two plugins that send pings to every world (world hopper plugin already does this). They suggested that if this is an important feature, it would require a change to the world hopper plugin to expose the pings it gets as a service. I'm following up with them to see what would be required and whether i can help with that

**Comment by 67protons** [comment]
Disappointing but sounds good. Ya, it'd be nice if the world hopper plugin ping values could just be queried by all other plugins.
